VAMobile/src/screens/HealthScreen/Appointments/AppointmentDetailsCommon/AppointmentAddressAndNumber.tsx

```diff
@@ -36,6 +37,86 @@ const AppointmentAddressAndNumber: FC<AppointmentAddressAndNumberProps> = ({ att
     return <></>
   }
 
+  const getAddress = () => {
+    return (
+      <>
+        {!!address?.street && (
+          <TextView variant="MobileBody" selectable={true} accessible={false}>
+            {address.street}
+          </TextView>
+        )}
+        {!!address?.city && address?.state && address?.zipCode && (
+          <TextView variant="MobileBody" selectable={true} accessible={false}>
+            {`${address.city}, ${address.state} ${address.zipCode}`}
+          </TextView>
+        )}
+      </>
+    )
+  }
+
+  if (appointmentType === AppointmentTypeConstants.VA && serviceCategoryName !== 'COMPENSATION & PENSION') {
```

Comment:
I noticed `COMPENSATION & PENSION` is referenced in many different files. It might be worth making a const for

Reply:
true, however the next set of appt work I will be able to condense a ton of things so I feel it is fine leaving it like this